const catchAndPrintError = error => {
  console.error(error);
  return null;
};

module.exports = { catchAndPrintError };
